package.json

```diff
@@ -22,9 +23,11 @@
   "dependencies": {
     "@types/sodium-native": "^2.3.5",
     "@types/sqlite3": "^3.1.6",
+    "ajv": "^6.12.3",
```

Comment:
This is a library that creates a validity function using json-schma and interface.